transform/transformers/pck_transformer.py

```diff
@@ -171,6 +176,27 @@ def preprocess_comments(self):
         self.data = data
         return self.data
 
+    def calculate_total_playback(self):
+        """
+        Calculates the total value for both acquisitions and proceeds from disposals for machinery and equipment section
+        as well as all sections.
+        q_code - 692 - Total value of all acquisitions questions.
+        q_code - 693 - Total value of all disposals questions.
+        q_code - 714 - Total value of all acquisitions questions for only machinery and equipments sections.
+        q_code - 715 - Total value of all disposals questions for only machinery and equipments sections.
+        """
+        if self.survey.get('survey_id') == self.qcas_survey_id:
+            all_acquisitions_questions = self.qcas_machinery_acquisitions_questions + self.qcas_other_acquisitions_questions
+
+            total_machinery_acquisitions = sum(Decimal(value) for q_code, value in self.data.items() if q_code in self.qcas_machinery_acquisitions_questions)
+            total_disposals = sum(Decimal(value) for q_code, value in self.data.items() if q_code in self.qcas_disposals_questions)
+            all_acquisitions_total = sum(Decimal(value) for q_code, value in self.data.items() if q_code in all_acquisitions_questions)
+
+            self.data['714'] = str(total_machinery_acquisitions)
+            self.data['715'] = str(total_disposals)
```

Comment:
This strikes me as a possible error if questions get added later on. 

If anyone adds other disposal questions and doesn't update this code to exclude them, they could get caught out.
On the other hand, this *should* become apparent through the tests.

Reply:
We'd get new requirements for any change like this. At the moment we shouldn't worry about it. This change is only needed for the old legacy system. Any future changes like this will go through a new system that will do this kind of work itself.